src/main/java/io/prometheus/wls/rest/domain/ExporterConfig.java

```diff
@@ -1,17 +1,23 @@
 package io.prometheus.wls.rest.domain;
 /*
- * Copyright (c) 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2020, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
I think we need to remove "All rights reserved" according to a review comment from Mark (https://github.com/oracle/weblogic-monitoring-exporter/pull/86#discussion_r395182828)

Reply:
yes that is right, any file you update in 2020, you should remove the "all rights reserved"

Reply:
Done.